comm.c

```diff
@@ -91,7 +91,7 @@ bool write_comm_request(struct swaylock_password *pw) {
 	result = true;
 
 out:
-	clear_password_buffer(pw);
+	explicit_bzero(pw, sizeof *pw);
```

Comment:
Style: please add parentheses around the `sizeof` argument

Reply:
Last project i worked on insisted to do sizeof without parenthesis. anyway. changed it to your liking.